src/app/js/formats/cartography/CartographyView.js

```diff
@@ -0,0 +1,170 @@
+        const value = this.props.chartData[data.properties.ISO_A3];
+        if (!value) {
+            return;
+        }
+        this.props.showTooltip({
+            origin: { x, y },
+            content: (
+                <dl>
+                    <dt>{data.properties.NAME}</dt>
+                    <dd>{value}</dd>
+                </dl>
+            ),
+        });
+    };
+    handleLeave = () => {
+        this.props.hideTooltip();
+    };
+    render() {
+        const { chartData, maxValue } = this.props;
+
+        return (
+            <div style={styles.container}>
+                <ComposableMap
+                    projectionConfig={projectionConfig}
+                    width={980}
+                    height={551}
+                    style={styles.composableMap}
+                >
+                    <ZoomableGroup disablePanning center={[0, 20]}>
+                        <Geographies disableOptimization geography={mapJson}>
+                            {(geographies, projection) =>
+                                geographies.map(
+                                    (geography, i) =>
+                                        geography.id !== 'ATA' && (
+                                            <Geography
+                                                key={`geography-${i}`}
+                                                onMouseMove={this.handleMove}
+                                                onMouseLeave={this.handleLeave}
+                                                cacheId={`geography-${i}`}
+                                                geography={geography}
+                                                projection={projection}
+                                                style={styles.geography({
+                                                    value:
+                                                        chartData[
+                                                            geography.properties
+                                                                .ISO_A3
+                                                        ] || 0,
+                                                    maxValue,
+                                                })}
+                                            />
+                                        ),
+                                )
+                            }
+                        </Geographies>
+                    </ZoomableGroup>
+                </ComposableMap>
+                <Tooltip />
+            </div>
+        );
+    }
+}
+
+CartographyView.propTypes = {
+    chartData: PropTypes.object.isRequired,
+    maxValue: PropTypes.number.isRequired,
+    showTooltip: PropTypes.func.isRequired,
+    hideTooltip: PropTypes.func.isRequired,
+};
+
+const mapStateToProps = (state, { chartData }) => {
+    if (!chartData) {
+        return {
+            chartData: {},
+            maxValue: 0,
+        };
+    }
+    return {
```

Comment:
I'm afraid this might get heavy. Consider using selectors (and reselect for memoization) if it's the case.